### DISCUSSION TITLE
mindless click as fast as possible

### DISCUSSION THREAD

**Discussion by EttoreZugliani** [Nain57/Smart-AutoClicker#215] [Q&A]
Hello, i'm using Smart AutoClicker for some incremental games and its working great for recognizing and clicking some screen events.
However a big part of this kind of games is to mindless click the same spot multiple times as fast as possible. Take cookie clicker for example, i wanted to click the cookie as many times per second as the software can register. Is it possible to achieve with Smart AutoClicker?

Thanks in advance.

**Comment by hatbrox** [comment]
just create an event with a condition that recognise the area you want to tap.
the event action is to tap on the condition
such event would just get triggered endlessly
(you could copy that action 20x times as it is faster than triggering the event 20x)

another solution is to use another a simpler auto-click app, many exist that don't use event, condition etc. just a stupid click for ever.
